state-chain/pallets/cf-ingress-egress/src/lib.rs

```diff
@@ -748,6 +753,13 @@ pub mod pallet {
 		FailedToRejectTaintedTransaction {
 			tx_id: <T::TargetChain as Chain>::DepositDetails,
 		},
+		UnknownPrimaryBroker {
+			broker_id: T::AccountId,
+		},
+		UnknownAffiliateBroker {
```

Comment:
```suggestion
		UnknownBroker {
			broker_id: T::AccountId,
		},
		UnknownAffiliate {
```

Do we need to introduce this new distinction between 'primary broker' vs 'affiliate broker'? Is broker/affiliate not enough?

Reply:
I'm not sure. To me "affiliate" is short for "affiliate broker", which means simply "broker" is ambiguous 🤷‍♂️. Not a problem if we stick to the convention, but there is also value in being explicit. No strong opinion though, I will make the change.